Jesse-Kramer/tumi-mundo
# Home page

## What does this change?
- Added the home page from the Figma design. 
- Added basic styling like fonts.
- Added fonts from google fonts.

Tried making it as semantic as possible. 
```HTML
<section>
    <article>
        <img src="/progress.svg" alt="Progress bar">
        <div class="progress-text">
            <p>Today's listening</p>
            <h1>00:00</h1>
            <p>Of your 5-minute goal</p>
        </div>
    </article>
    <article>
        <img src="/characters/rat.svg" alt="Rat character"/>
    </article>
</section>
```

## Figma design
![image](https://github.com/user-attachments/assets/00835f1a-099c-45be-b90a-cbe75d0df504)


## Result
![image](https://github.com/user-attachments/assets/b212534b-2d8f-4bdb-bf57-1729430296d9)

## Whats needs to be done in the future?
- Make the pop-up above the character. Wanted to wait for a fully functional site, maybe we can connect the pop-up to some data.
![image](https://github.com/user-attachments/assets/3383c89f-0270-46a2-b19e-5b403124e34f)
- Change the background (window, carpet etc.) to svg instead of PNG.
- In the future connect the choosen buddy with the data from the choosen profile.
- Connect the progress bar with data.



## How Has This Been Tested?


## Images
![image](https://github.com/user-attachments/assets/e5b9d870-184d-4ea0-9713-0e090f3db159)
![image](https://github.com/user-attachments/assets/bfaf553c-380b-4b3e-8a31-514ea97a5887)
![image](https://github.com/user-attachments/assets/ec182a5d-e611-4037-977c-af98db880aac)




## How to review
1. To start the project you firstly have load the repo from Github Desktop.
2. Navigate to the branch of this pull-request
3. Open the terminal and enter `npm install` to install the dependencies.
4. To start the project enter `npm run dev` to start the project. Now you will be redirected to "[https://localhost:5173/](https://localhost:5173/)".

"On the home page I want to see the time that i've been listening today."
"On the home page I want to see my buddy."
